usr/share/rear/layout/prep-for-mount/default/600_show_unprocessed.sh

```diff
@@ -0,0 +1,58 @@
+
+# Warn about missing components and for each missing component
+# offer the user a way to manually add code that mounts it.
+
+rear_workflow="rear $WORKFLOW"
+rear_shell_history="$( echo -e "vi $LAYOUT_CODE\nless $LAYOUT_CODE" )"
+
+unset choices
+choices[0]="View $LAYOUT_CODE"
+choices[1]="Edit $LAYOUT_CODE"
+choices[2]="Go to Relax-and-Recover shell"
+choices[3]="Continue '$rear_workflow'"
+choices[4]="Abort '$rear_workflow'"
+
+while read status name type junk ; do
+    missing_component="$name ($type)"
+    LogUserOutput "No code has been generated to mount $missing_component."
+    LogUserOutput "To mount $missing_component manually add code to $LAYOUT_CODE or abort."
+    while true ; do
+        # The default user input is "Continue" to make it possible to run ReaR unattended
+        # so that 'rear recover' proceeds after the timeout regardless that it probably fails
```

Comment:
Tiny copy&paste typo in comment:
```
# so that 'rear recover' proceeds ...
```
must be
```
# so that 'rear repair' proceeds
```
here.

Reply:
Changes committed.